packages/tinymce/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "@web-stories-wp/tinymce",
+  "private": true,
+  "engines": {
+    "node": ">= 15",
+    "npm": "~7"
+  },
+  "exports": {
+    "default": "./src/web-stories.js"
+  },
+  "main": "./src/web-stories.js",
```

Comment:
Let's rename the `./src/web-stories.js` file to `./src/index.js` for consistency with other packages' entry points.